spec/system/organisations/students/show_spec.rb

```diff
@@ -250,14 +250,38 @@
 
       before { org_admin_user.update!(organisation: organisation) }
 
-      scenario 'org admin cannot see private notes' do
+      scenario "org admin cannot see private notes" do
         sign_in_user org_admin_user,
                      referrer: org_student_path(student_org_admin)
 
         expect(page).to have_text(org_admin_user.name)
-        expect(page).not_to have_text('Notes')
+        expect(page).not_to have_text("Notes")
         expect(page).not_to have_text(coach_note.note)
       end
     end
+
+    context "org admin checks feedback on student submissions" do
+      let(:submission) do
+        create :timeline_event,
+               :evaluated,
+               :passed,
+               :with_owners,
+               latest: true,
+               target: target_l2,
+               owners: [student]
+      end
+      before do
+        4.times do
+          create :startup_feedback,
+                 timeline_event_id: submission.id,
+                 faculty_id: faculty.id
+        end
+      end
+      scenario "org admin can see feedback on student submissions" do
+        sign_in_user org_admin_user, referrer: timeline_event_path(submission)
+
+        expect(page).to have_selector(".feedback_item", count: 4)
```

Comment:
The main change you're making in this PR is to `timeline_events/show.html.erb`, so why are the tests are being added to `organisations/students/show_spec.rb` instead of `spec/system/submissions/show_spec.rb`?

`submissions/show_spec.rb` tests the functionality of the `TimelineEvent#show` page in detail; that's the best place to test added features for this page.

### 1. Move specs to the right place

Let's update `submissions/show_spec.rb` to check for the presence of feedback in the page as well all the other info already tested. You can actually do this part using the role of the student which is what is mainly used in `submissions/show_spec.rb`.

### 2. Keep the complexity of test context to a minimum

Your context sets up **four** feedback items. You don't _need_ four - you can run all your tests with just two `startup_feedback` entries - so that'll do.

### 3. Test for presence of content, not presentation

Unless you absolutely have to, never check for _presentational_ elements such as `.feedback_item`. We don't care about how the page is styled / what classes are used to do so. When we're writing tests, we're most often concerned about the _contents_ of the page rendered.

Check whether the text of both feedback items have been shown in the page.

### 4. Tests bug-fixes first.

This PR actually fixes a bug in the `TimelineEventPolicy` class which caused the submission page to not actually render for org admins. This bug went to production because we didn't actually test the original functionality of this page by visiting it as an org admin.

In `submissions/show_spec.rb`, vist the page as an org admin user and check whether `submission.title` is visible in the response.

This should be enough, as other specs check the full functionality of the page, and what we're testing in this new access check is whether an org admin user can access this page. Make sure that this new test _fails_ when the corresponding fix (in `app/policies/timeline_event_policy.rb:25`) isn't present.

Reply:
Tested `TimelineEventPolicy` with and without the fix.